core/builtins/native/kotlin/Iterator.kt

```diff
@@ -43,6 +83,15 @@ public trait MutableListIterator<T> : ListIterator<T>, MutableIterator<T> {
 
     // Modification Operations
     override fun remove(): Unit
+
+    /**
+     * Replaces the last element returned by [next] or [previous] with the specified element [e].
+     */
     public fun set(e: T): Unit
+
+    /**
+     * Adds the specified element [e] into the underlying collection immediately before the element that would be
+     * returned by [next].
```

Comment:
Will it affect what is returned next? Unclear from the text. What if previous was called?